x-pack/plugins/rollup/public/crud_app/services/jobs.js

```diff
@@ -0,0 +1,154 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+function removeEmptyValues(object) {
+  Object.keys(object).forEach(key => {
+    if (object[key] == null || object[key].trim() === '') {
```

Comment:
I think relying on the values in `object` to be either undefined, null, or a string is slightly risky. `String(object[key]).trim() === ''` would probably be a bit safer.